### DISCUSSION TITLE
Question about simultaneous use of 2 different versions of the OneDrive client

### DISCUSSION THREAD

**Discussion by jrpelegrina** [abraunegg/onedrive#3026] [General]
First of all, congratulations on the work you are doing

I am going to start testing the client version (2.5.3) and reading the documentation I see that you indicate that the same OneDrive account should not be synchronized simultaneously using this client and the old versions (2.4.x).
What would happen if for some reason they were used?
Regards

**Comment by abraunegg** [comment]
Please search discussions where that has been explained a number of times.

Key points:
* v2.5.x is a complete re-write ... 99% of everything operates *differently*
* All the file integrity handling is different

> What would happen if for some reason they were used?

You may experience data loss - either under v2.4.x or v2.5.x

**Comment by jrpelegrina** [comment]
OK, I'll keep that in mind
Thank you very much for your quick response
